test/core/storage/repository/in_memory_repository_test.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "storage/repository/impl/in_memory_repository.hpp"
+
+#include "storage/repository/repository_error.hpp"
+#include "testutil/literals.hpp"
+#include "testutil/outcome.hpp"
+#include "testutil/storage/base_fs_test.hpp"
+
+using fc::sector_storage::stores::LocalPath;
+using fc::sector_storage::stores::StorageConfig;
+using fc::storage::repository::InMemoryRepository;
+
+class InMemoryRepositoryTest : public test::BaseFS_Test {
+ public:
+  InMemoryRepositoryTest() : test::BaseFS_Test("in_memory_repository_test") {}
+
+  /**
+   * Create a test repository with an empty file.
+   */
+  void SetUp() override {
```

Comment:
This override is unnecessary, because parent `SetUp` will be called if it's there is no override, please remove